x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/expandable_flyout/alert_details_right_panel_json_tab.cy.ts

```diff
@@ -16,7 +16,8 @@ import { getNewRule } from '../../../../objects/rule';
 import { ALERTS_URL } from '../../../../urls/navigation';
 import { waitForAlertsToPopulate } from '../../../../tasks/create_new_rule';
 
-describe(
+// Failing constantly on CI
+describe.skip(
```

Comment:
are we sure this is still constantly failing on CI?

There was an issue introduced by this [PR](https://github.com/elastic/kibana/pull/164918) 4 days ago (because we don't run all Cypress tests on all PRs, but let's not start on that topic...), but it was fixed shortly after (see this [PR](https://github.com/elastic/kibana/pull/164016) and more specifically this line).
Looking at a few builds over the last couple of days, I'm not seeing any failures for this test, if I'm not mistaken?

Reply:
reverted, Thank you for the update @PhilippeOberti 🙇

Reply:
awesome thank you, PR approved :)

Reply:
@PhilippeOberti could you take a look? https://buildkite.com/elastic/kibana-pull-request/builds/154764#018a45af-24c8-422f-bbe6-4efcb9d8c064